images/macos/provision/core/aws.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash -e -o pipefail
 
+source ~/utils/utils.sh
+
 echo Installing aws...
-curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
+AWSCLIURL="https://awscli.amazonaws.com/AWSCLIV2.pkg"
```

Comment:
could you please add underscores to variable name? like AWS_CLI_URL to make it more readable

Reply:
Done